dotnet/src/webdriver/SeleniumManager.cs

```diff
@@ -34,6 +35,8 @@ namespace OpenQA.Selenium
     /// </summary>
     public static class SeleniumManager
     {
+        private static readonly ILogger _logger = Log.ForContext.GetLogger(typeof(SeleniumManager));
```

Comment:
In static classes I think it is better not to store `ILogger` in a field, but get logger dynamically right in the place where you do logging (before `_logger.Trace(...)`). Because currently only the first occurred `LogContext`'s logger will be written to the field and used in all cases.

Reply:
I reworked it, in general we want to use `private static readonly` in all cases. It guarantees that class captures `ILogger` one time. Then log's emission happens later, based on current log context.